bin/deploy-prod

```diff
@@ -11,8 +11,6 @@ TIMESTAMP=$(date +%s)
 # which has the side effect of recreating all the containers.
 # That's kind of messy.  We'd prefer to use a secret manager.  We'll set that up next.
 DBUSERNAME=uat
```

Comment:
**Suggestion** (blocking): Please remove/update the comments here if they are out-dated by this change. Also consider removing `DBUSERNAME` as well b/c it is no longer passed in (maybe it should be?).

Reply:
Ha, thank you!